components/camel-azure-storage-datalake/src/main/java/org/apache/camel/component/azure/storage/datalake/operations/DataLakeDirectoryOperations.java

```diff
@@ -0,0 +1,33 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.apache.camel.component.azure.storage.datalake.operations;
+
+import org.apache.camel.component.azure.storage.datalake.DataLakeConfiguration;
+import org.apache.camel.component.azure.storage.datalake.DataLakeConfigurationOptionsProxy;
+import org.apache.camel.component.azure.storage.datalake.client.DataLakeDirectoryClientWrapper;
+import org.apache.camel.util.ObjectHelper;
+
+public class DataLakeDirectoryOperations {
```

Comment:
Is there any good use for this class?

Reply:
So, there are some operations on directory as well in datalake gen2. I am gonna add those in here.